packages/shared/models/board.ts

```diff
@@ -0,0 +1,34 @@
+import mongoose from 'mongoose'
+
+const boardSchema = new mongoose.Schema({
+  id: String,
+  title: String,
+  description: String,
+  createdAt: Date,
+  status: String,
+  users: [
+    {
+      id: String,
+      username: String,
+      avatarUrl: String
+    }
+  ],
+  columns: [
+    {
+      id: String,
+      title: String,
+      tasks: [
+        {
+          id: String,
+          title: String,
+          captionUrl: String,
```

Comment:
what is this field for?

Reply:
This field is for board background image.